frontend/public/components/edit-yaml.jsx

```diff
@@ -666,7 +666,7 @@ const EditYAML_ = connect(stateToProps)(
       });
 
       const { error, success, stale, yaml, height, showSidebar } = this.state;
-      const { obj, download = true, header } = this.props;
+      const { obj, download = true, header, showReload = true } = this.props;
```

Comment:
Booleans that default to true are confusing. I'd make this `noReload`.

Reply:
changed to `nonK8s`, but seems a little wonky specifing `{stale && !nonK8s && ( ...`

Reply:
> changed to `nonK8s`, but seems a little wonky specifing `{stale && !nonK8s && ( ...`

Is `stale` ever true if this isn't a k8s object? I wonder if the `nonK8s` check is needed

Reply:
for `stale` the `nonK8s` check isn't necessary.  Still have ` {!create && !nonK8s && (..` - I just don't like '!non...`.

Reply:
You could change `nonK8s` to something like `anyYAML` or `genericYAML`. I think that's more clear anyway.

Reply:
We should be skipping all the logic that checks for schemas, etc., when set.

Reply:
I like it, changed to `genericYAML` -thanks